### PR TITLE
Add hover to UI Elements

### PR DESCRIPTION
**Pull request by rubenmoya**
Related to: https://github.com/CartoDB/cartodb/issues/13074

### CODE REVIEW

File: app/assets/stylesheets/editor-3/_block-list.scss [hunk 1]
```diff
@@ -145,36 +158,44 @@
...
+    }
+  }
+
+  &.is-selected {
+    color: $cWhite;
+  }
+
+  &:first-child::before {
+    display: none;
+    background: rd;
```
Comment (matallo) [inline app/assets/stylesheets/editor-3/_block-list.scss]: wtf? 😂

File: app/assets/stylesheets/editor-3/_custom-carousel.scss [hunk 2]
```diff
@@ -44,52 +46,62 @@
...
-    border: 1px solid $cMainLine;
+    border: 2px solid $cBlueHover;
     cursor: pointer;
   }
+
+  &.is-selected {
+    border: 2px solid $cBlackHover;
+  }
+
+  &--small {
```
Comment (matallo) [inline app/assets/stylesheets/editor-3/_custom-carousel.scss]: same than ^^^

in these cases I think it is preferable a new selector

File: app/assets/stylesheets/editor-3/_custom-carousel.scss [hunk 3]
```diff
@@ -44,52 +46,62 @@
...
+  opacity: 0;
   z-index: 100;
+  pointer-events: none;
+
+  &.is-visible {
+    opacity: 1;
+  }
+
+  &--left {
+    @include background-horizontal(#FFF, rgba(255, 255, 255, 0));
```
Comment (matallo) [inline app/assets/stylesheets/editor-3/_custom-carousel.scss]: I know it's not the ultimate goal for this PR but can't we get rid of gradient mixins? in any case new selector and `rgba(#fff, 0)`

**Comment by matallo** [comment]
focus states are slightly lighter than hover states, also something looks weird in fields with multiple inputs

**Comment by piensaenpixel** [comment]
Love it, only two things:

- If I select a dataset, also the layer has the hover outline, I know it's the actual behavior but with this change of color I think it's a bit weird
image

- Could it possible consolidate the fade transition in all the elements? input, carousel, etc…

**Comment by rubenmoya** [comment]
About the first point, I'm not sure if that's possible since if you're hovering the child you're hovering the parent too. Maybe creating a `::before` with the parent's size and creating a border on top of the parents one with the color we want... 

About the second point we've already talked about consolidating it, some elements are using `0.25s`, others `0.3s`... we should pick a delay and stick to it, any advice on what delay would be nice?

**Comment by matallo** [comment]
it could be done in a similar way than we use the mouse actions in this other PR https://github.com/CartoDB/cartodb/pull/13102/files#diff-dabcc2a116965bdb8786b7b369bd880a

in any case I'd add a new issue for the improvement, we're just about to deploy this

**Pull request merged**